packages/contact-center/station-login/src/station-login/custom-elements.d.ts

```diff
@@ -0,0 +1,23 @@
+// This is used to let know typescript that the custom element is available in the global scope.
```

Comment:
### This is used to let know typescript that the custom element is available in the global scope.